apps/condo/lang/ru/ru.json

```diff
@@ -816,6 +816,18 @@
   "notification.messages.TICKET_ASSIGNEE_CONNECTED.push.title": "Новая заявка",
   "notification.messages.TICKET_EXECUTOR_CONNECTED.email.subject": "Вы назначены исполнителем(-ницей) по заявке №{data.ticketNumber}",
   "notification.messages.TICKET_EXECUTOR_CONNECTED.push.title": "Новая заявка",
+  "notification.messages.TICKET_STATUS_OPENED.email.subject": "Привет, это Doma! \uD83E\uDD96",
+  "notification.messages.TICKET_STATUS_OPENED.push.title": "Привет, это Doma! \uD83E\uDD96",
+  "notification.messages.TICKET_STATUS_IN_PROGRESS.email.subject": "У нас новости по заявке №{data.ticketNumber}! \uD83E\uDD96",
+  "notification.messages.TICKET_STATUS_IN_PROGRESS.push.title": "У нас новости по заявке №{data.ticketNumber}! \uD83E\uDD96",
```

Comment:
Please make sure push `title` value fits within 25 chars, in other way it would be truncated by iOS